x-pack/plugins/apm/server/index.ts

```diff
@@ -52,7 +51,17 @@ const configSchema = schema.object({
 
 // plugin config
 export const config: PluginConfigDescriptor<APMXPackConfig> = {
-  deprecations: ({ deprecate }) => [deprecate('enabled', '8.0.0')],
+  deprecations: ({ deprecate, renameFromRoot }) => [
+    deprecate('enabled', '8.0.0'),
+    renameFromRoot(
```

Comment:
@ogupte I think you can use `renameFromRoot` when working on https://github.com/elastic/kibana/issues/108160